src/olympia/migrations/1023-remove-collections-users.sql

```diff
@@ -0,0 +1,8 @@
+SET @to_group := (
+    SELECT `id` FROM `groups` WHERE `name` = 'Featured Themes Contributors');
+-- 2143965 is the id of the Featured Themes Collections. We want all its
+-- contributors to be part of the 'Featured Themes Contributors' group.
+INSERT IGNORE INTO `groups_users` (`group_id`, `user_id`) (
+    SELECT @to_group, `user_id` FROM `collections_users`
+    WHERE `collection_id` = 2143965);
+DROP TABLE `collections_users`;
```

Comment:
Can we get the list of user-ids from ops before running this? Unfortunately, this table isn't available in our Redash. Just for safe-keeping if MySQL get's destroyed while running this.

Reply:
Well tbf the table data is already useless now that the contributors feature has been removed :>